protostar/cli/network_command_util.py

```diff
@@ -93,6 +93,6 @@ def _normalize_chain_id(
             return StarknetChainId(arg)
         except ValueError as ex:
             raise ProtostarException(
-                "Invalid chain id value.\n"
+                f"Invalid chain id value: {arg}.\n"
```

Comment:
A bit unrelated but I decided we could maybe squash it in here. No big deal anyhow, I think it's good to show what the invalid value is, it makes debugging easier/faster.